src/Symfony/Component/Process/ProcessBuilder.php

```diff
@@ -221,8 +221,10 @@ public function getProcess()
 
         $options = $this->options;
 
-        $arguments = $this->prefix ? array_merge(array($this->prefix), $this->arguments) : $this->arguments;
-        $script = implode(' ', array_map(array(__NAMESPACE__.'\\ProcessUtils', 'escapeArgument'), $arguments));
+        $prefix = $this->prefix ? $this->prefix : '';
```

Comment:
No :) You can use it [since PHP 5.3](http://php.net/manual/en/language.operators.comparison.php#language.operators.comparison.ternary):

> Since PHP 5.3, it is possible to leave out the middle part of the ternary operator. Expression expr1 ?: expr3 returns expr1 if expr1 evaluates to TRUE, and expr3 otherwise.